mysql-test/suite/galera/t/MW-258.test

```diff
@@ -28,12 +31,20 @@ SHOW STATUS LIKE 'wsrep_desync_count';
 SHOW VARIABLES LIKE 'wsrep_desync';
 UNLOCK TABLES;
 
+--connect node_1c, 127.0.0.1, root, , test, $NODE_MYPORT_1
+SET GLOBAL wsrep_provider_options = 'signal=wsrep_desync_left_local_monitor';
```

Comment:
We should use script instead.
--source include/galera_clear_sync_point.inc
--source include/galera_signal_sync_point.inc